examples/demo.ts

```diff
@@ -6,9 +6,8 @@ import Cloudflare from 'cloudflare';
 const client = new Cloudflare();
 
 async function main() {
-  const accountId = '<your accound ID>';
-
-  const response = await client.ai.run(accountId, '@cf/meta/llama-2-7b-chat-int8', {
+  const response = await client.workers.ai.run('@cf/meta/llama-2-7b-chat-int8', {
+    account_id: '<your account ID>',
```

Comment:
```suggestion
    account_id: process.env['CLOUDFLARE_ACCOUNT_ID'],
```

unless there's an account ID that cloudflare is comfortable exposing publicly as a demo ID? @jacobbednarz ?

Reply:
in the demos, i'd rather lean on environment variables and checks to allow people to copy/paste them and know they work as they should.

account/zone ids aren't sensitive so i can add the example to GitHub Actions later today for this to work.